packages/react-core/src/scalprum-component.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, Suspense, useState, ReactNode, useReducer } from 'react';
+import React, { useEffect, Suspense, useState, ReactNode, useReducer, useMemo } from 'react';
```

Comment:
`useMemo` is not used
```suggestion
import React, { useEffect, Suspense, useState, ReactNode, useReducer } from 'react';
```